sdk/common/env.go

```diff
@@ -0,0 +1,59 @@
+package common
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+//EnvVar provides API for access to env variable
+type EnvVar string
+
+func (v EnvVar) String() string {
+	return fmt.Sprintf("%v:%v", v.Name(), v.StringValue())
+}
+
+//StringValue returns value of env variable as string
+func (v EnvVar) StringValue() string {
+	return os.Getenv(v.Name())
+}
+
+//GetOrDefaultStringListValue returns list of string separated by space or if env variable have not a value returns default value
+func (v EnvVar) GetOrDefaultStringListValue(defaultValue []string) []string {
```

Comment:
Please consider rename it to `GetStringListValueOrDefault`, sounds a bit better to me

Reply:
thanks, fixed